shakenfist/config.py

```diff
@@ -137,6 +137,8 @@ def parse(self):
                     value = int(value)
                 elif isinstance(CONFIG_DEFAULTS[flag], float):
                     value = float(value)
+                elif isinstance(CONFIG_DEFAULTS[flag], bool):
+                    value = (value == 'true')
```

Comment:
Given that some people may use `True`, maybe
```
from distutils.util import strtobool
value = strtobool(value)
```

Reply:
Done.